### PR TITLE
Task#93 shed-.jsp menu

### PR DESCRIPTION
**Pull request by HrBjarup**
Prototype version of the shed menu

### CODE REVIEW

File: FogCarport/src/main/webapp/header.jsp [hunk 1]
```diff
@@ -24,7 +24,7 @@
         <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"  crossorigin="anonymous">
 
         <!-- Custom stylesheets -->
-        <link href="css/login.css" rel="stylesheet" type="text/css"/>
+        <!--<link href="css/login.css" rel="stylesheet" type="text/css"/>-->
```
Comment (MalteMagnussen) [inline FogCarport/src/main/webapp/header.jsp]: Ah, nice, I had forgotten about that one. Thanks.

**Comment by HrBjarup** [comment]
image

**Comment by HrBjarup** [comment]
image

**Comment by HrBjarup** [comment]
image

**Comment by HrBjarup** [comment]
image

**Pull request merged**